hack/codegen.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+set -eux
+
+IS_CONTAINER=${IS_CONTAINER:-false}
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-podman}"
+
+if [ "${IS_CONTAINER}" != "false" ]; then
+  export XDG_CACHE_HOME=/tmp/.cache
+  mkdir /tmp/unit
+  cp -r ./* /tmp/unit
+  cd /tmp/unit
+  make generate 
```

Comment:
that's a good starting point. Here we need to compare the existing (in /go/src/github.com/metal3-io/cluster-api-provider-metal3) to what was generated and fail if there is a difference

Reply:
Handled in codegen.sh